package.json

```diff
@@ -6,12 +6,10 @@
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
-    "format": "prettier --write **/*.ts",
-    "format-check": "prettier --check **/*.ts",
     "lint": "eslint src/**/*.ts",
     "package": "ncc build --source-map --license licenses.txt",
     "test": "jest",
-    "all": "npm run build && npm run format && npm run lint && npm run package && npm test"
+    "all": "npm run build && npm run lint && npm run package && npm test"
```

Comment:
I'll have to double check this one, the original order here came from Github's template project.

Reply:
It looks to me like they're running them in alphabetical order, whereas I think it makes sense to run them in logical gating order.

Reply:
I tested it out, and I need to do the build before the test. The tests include a test of the built typescript files in the lib folder. It uses the built files because it is loading them directly into node to verify they can handle the standard in communication model of GH actions.